projects/js-packages/charts/package.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "@automattic/jetpack-charts",
```

Comment:
I guess it should be @automattic/charts here too

Reply:
```suggestion
	"name": "@automattic/charts",
```